configs/sst_middleware-tomcat.yaml

```diff
@@ -0,0 +1,22 @@
+document: feedback-pipeline-unwanted
```

Comment:
It looks like you're making a workload so changing the document type to `feedback-pipeline-workload` should fix the failing CI checks.

Reply:
Ah, I was staring at the CI failure in confusion, thanks a lot @bstinsonmhk . Repushed like that.